impl-endpoints/cloudflare-go/runner.go

```diff
@@ -235,15 +286,16 @@ func main() {
 		os.Exit(64)
 	}
 
+	log.SetFlags(0)
 	if *role == "client" {
 		if err := doClient(handler); err != nil {
-			log.Fatal("Error: ", err)
+			log.Fatal(err)
 		}
 	} else if *role == "server" {
 		if err := doServer(handler); err != nil {
-			log.Fatal("Error: ", err)
+			log.Fatal(err)
 		}
 	} else {
-		log.Fatalf("unknown role \"%s\"", *role)
+		log.Fatalf("Unknown role \"%s\"", *role)
```

Comment:
This makes me wonder if we should have a sentinel return value for invalid parameters. @xvzcf, thoughts?

Reply:
Will address this in the follow-up PR.

Reply:
See #39, though I'm not sure this totally addresses the point!